### PR TITLE
Major refactoring of release posts

### PR DESCRIPTION
**Pull request by mikewalch**
- Combined all posts about each release into one
- The 'release-notes' category no longer exists
- Removed all 'blog' posts about releases.
- The blog archive nows shows all releases and was renamed news.

### CODE REVIEW

File: _posts/blog/2016-01-11-webindex-long-run.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title:  "Running Webindex for 3 days on EC2"
-date:   2016-01-11 03:30:00
+date:   2016-01-11 15:30:00
```
Comment (keith-turner) [inline _posts/blog/2016-01-11-webindex-long-run.md]: This makes the links to 01/10 not work.

File: _posts/release/2016-09-27-fluo-1.0.0-incubating.md [hunk 2]
```diff
@@ -1,27 +1,139 @@
...
- * Verify release: [ASC][asc] [MD5][md5] [SHA1][sha1]
- * [Release notes][notes]
- * [Documentation][docs]
- * [Javadocs][apidocs] 
+ * Download a release tarball and verify by these [procedures] using these [KEYS]
  
-Starting with 1.0.0-incubating, Apache Fluo will follow [semver](http://semver.org/) for all API
-changes. The API consists of everything under the **org.apache.fluo.api** package. Code outside of this
+   | [fluo-1.0.0-incubating-bin.tar.gz][binary] | [ASC][asc] [MD5][md5] [SHA1][sha1] |
+   | [fluo-1.0.0-incubating-source-release.tar.gz][src-release] | [ASC][src-asc] |
```
Comment (keith-turner) [inline _posts/release/2016-09-27-fluo-1.0.0-incubating.md]: could have hash links here

File: _posts/blog/2016-01-11-webindex-long-run.md (hunk 1 above)
Comment (ctubbsii) [reply _posts/blog/2016-01-11-webindex-long-run.md]: This is apparently a known issue: jekyll/jekyll#1069

Comment (ctubbsii) [reply _posts/blog/2016-01-11-webindex-long-run.md]: We should just always put the time zone, to reduce any ambiguity.

**Review by keith-turner — APPROVED** [review]
I like these changes.  Makes the site easier to navigate and understand.

**Pull request merged**